### ISSUE TITLE
Cannot find namespace Instabug in VS2015 (Windows)

### ISSUE THREAD

**Issue by Fonz001** [Instabug/instabug-xamarin#2]
Hi,

I followed all instructions in the README (Android works ! :+1: ). However, I could not get iOS to work. I'm working from a Windows machine (remote build) with the following version of Visual Studio:

Microsoft Visual Studio Community 2015
Version 14.0.25425 Update 3

I follow all instructions in the README (import project, reference project from iOS application project). However, when I try to refence the Instabug namespace, I get the following error:

Code: `Instabug.Instabug.StartWithToken(@"", Instabug.IBGInvocationEvent.IBGInvocationEventFloatingButton);`
Error: `Error CS0103 The name 'Instabug' does not exist in the current context`

So I double checked that i built the InstabugIOSSDK project. And I looked at what the Object browser says about the project (see image)

![screen](https://cloud.githubusercontent.com/assets/2063438/18625984/6a90930e-7e52-11e6-90f4-701421cc4e21.png)

So it seems that the problem is with the build of the InstabugIOSSDK project. However, now I'm stuck ... ;-)

Any ideas?

**Comment by Fonz001** [comment]
I have solved the problem (at least for my project). It is too much to write and test all steps, but a quick summary:

1. I created another Xamarin iOS library project (via Xamarin Studio, OSX) and pretty much copy-pasted everything from the InstabugIOSSDK project (including some tags from the .csproj file).  

2. I built the new project (success)

3. I referenced the new project from the iOS app project

4. The error ` Error CS0103 The name 'Instabug' does not exist in the current context` was gone. However. The enum: `IBGInvocationEvent` could still not be found.

5. I figured that there could be a difference between a real build and an IDE build, so maybe the iOS app project would build despite the missing enum...

6. I tried building the iOS app project. And voila! Success.

I'm leaving it up to the Instabug team to close this issue (maybe update the project? maybe integrate in NuGet (if possible)?).

( Cheers for the Instabug team, great product 👍 )
